apps/api/package.json

```diff
@@ -0,0 +1,60 @@
+{
+  "name": "@latitude-data/api",
+  "version": "0.0.1",
+  "scripts": {
+    "prettier": "prettier --write \"**/*.{ts,tsx,md}\"",
+    "build": "npx ts-node build.ts",
+    "lint": "eslint --ext .ts src/",
+    "lint:tests": "eslint --ext .ts test/",
+    "start": "node -r module-alias/register ./dist --env=production",
+    "dev": "tsx src/index.ts",
+    "tc": "tsc --noEmit",
+    "test": "vitest --run",
+    "test:watch": "vitest "
+  },
+  "packageManager": "pnpm@8.6.3",
+  "_moduleAliases": {
```

Comment:
What's this?

Reply:
module-alias package resolves ts's path alias at runtime so that running the production build works appropiately

Reply:
I think something that must be as perf as possible should be transpiled to JS and avoid as much as possible TS

Reply:
this is all js. To give more details:
- we compile to JS for production build with tsc
- tsc does not change code, it just transpiles, and since we use custom alias paths "@" these end up in the final js build
- since @ is not a valid path for node, this package modifies the node resolution algo at boot time in order to make it support our custom alias paths.

In summary, we trade some milliseconds at boot time in order to not have to deal with vite/rollup builds.